maquetado/application.scss

```diff
@@ -0,0 +1,3 @@
+.title {
+    color: red;
```

Comment:
Remember that the indentation should be of 2 spaces. Check this [style guide](https://github.com/Wolox/tech-guides/blob/master/frontend/docs/css-style-guide.md#indentation)